exercises/practice/robot-name/package.wren

```diff
@@ -0,0 +1,14 @@
+import "wren-package" for WrenPackage, Dependency
+import "os" for Process
+
+class Package is WrenPackage {
+  construct new() {}
+  name { "exercism/robot-simulator" }
```

Comment:
Wrong package name?

Reply:
A few other exercises with this bug: another PR is on the way.

Reply:
Or, should I just run the `update_testie.sh` script?

Reply:
Probably shouldn't get in the habit - that updates ALL the exercises.  I'm about to do it myself for a PR right before launch.  But in general maintenance I believe we agreed that's NOT what we typically want to do.